config/jobs/kubernetes/kops/kops-config.yaml

```diff
@@ -81,7 +81,7 @@ presubmits:
         - --aws-cluster-domain=test-cncf-aws.k8s.io
         - --check-leaked-resources=false
         - --cluster=
-        - --extract=ci/latest
+        - --extract=release/latest-1.18
```

Comment:
```suggestion
        - --extract=release/latest
```

Reply:
good eye, thats one less update to make every minor release